storage_devices/disk.py

```diff
@@ -227,16 +226,24 @@ def get_unplug_path(cls, device_id):
     def get_pci_address(cls, device_id):
         return TestRun.executor.run(f"cat /sys/block/{device_id}/device/address").stdout
 
+    @classmethod
+    def identify(cls, device_name: str) -> bool:
+        output = TestRun.executor.run(
+            f"realpath /sys/block/{device_name}/device/driver | grep scsi"
+        )
+        return output.exit_code == 0
+
 
 class SataDisk(Disk):
-    plug_all_command = "for i in $(find -H /sys/devices/ -path '*/scsi_host/*/scan' -type f); " \
-                       "do echo '- - -' > $i; done;"
+    plug_all_command = (
+        "for i in $(find -H /sys/devices/ -path '*/scsi_host/*/scan' -type f); "
+        "do echo '- - -' > $i; done;"
+    )
 
     def __init__(self, path, disk_type, serial_number, block_size):
         Disk.__init__(self, path, disk_type, serial_number, block_size)
         self.plug_command = SataDisk.plug_all_command
-        self.unplug_command = \
-            f"echo 1 > {self.get_unplug_path(self.get_device_id())}"
+        self.unplug_command = f"echo 1 > {self.get_unplug_path(self.get_device_id())}"
         self.pci_address = SataDisk.get_pci_address(self.get_device_id())
```

Comment:
`pci_address` is only an intermediate step to resolving unplug path, so keeping it as a filed is redundant for `SataDisk`